packages/base/card-api.gts

```diff
@@ -116,13 +116,15 @@ export interface CardContext {
   cardComponentModifier?: typeof Modifier<{
     Args: {
       Named: {
-        card: CardDef;
+        card?: CardDef;
+        cardId?: string;
```

Comment:
The prerendered cards don't come with `card`– there's only the card id (url). The reason I kept both `card` and `cardId` was for optimization purposed - if we have a `card` then we don't have to load it when the user wants to put it on the stack, delete it, or copy it (in the index card). Perhaps we can add a better type that says: "either card or cardId")